src/components/Search/Search.jsx

```diff
@@ -89,15 +90,19 @@ class Search extends Component {
   }
 }
 
+Search.defaultProps = {
+  placeholder: 'Search...',
+};
+
 Search.propTypes = {
   clearable: PropTypes.bool,
   clearIconUrl: PropTypes.string,
   disabled: PropTypes.bool,
-  iconUrl: PropTypes.string.isRequired,
+  iconUrl: PropTypes.string,
   instructionText: PropTypes.string,
-  onChange: PropTypes.func.isRequired,
+  onChange: PropTypes.func,
```

Comment:
now that onChange is not required, we should do a check at L32

Reply:
fixed